packages/fuels-abigen-macro/tests/test_projects/logging/Forc.lock

```diff
@@ -0,0 +1,11 @@
+[[package]]
```

Comment:
Don't commit the `Forc.lock` file

Reply:
Can we add a higher-level `.gitignore` to avoid this? Doesn't have to be at the root of the repo necessarily.